assignments/objects.js

```diff
@@ -18,28 +18,65 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const Kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
```

Comment:
```suggestion
speak: function() {
    return `Hello, my name is ${this.name}!`;
  }
}
```
You can also declare the method inside the object. The way you are doing it below also works, this is just a little better for the next developer, everything is in the same place. Same for Antionetta